README.md

```diff
@@ -14,14 +14,12 @@ A flexible JSON linter with out of the box support for OpenAPI v2 and v3.
 - Create custom rules to lint JSON or YAML objects
```

Comment:
Could you move the content of this file to something like `docs/intro.md` file, and update `README.md` to be a symbolic link that points to that intro file? Platform reads from `docs` folder only, so this will make it so that this introduction shows up in the published docs. And github will still render the readme on the repo landing just fine.

Reply:
It's been a while since I tried symlinks in git but I seem to remember it not being a barrel of fun. Have you done this yourself in other projects?

Reply:
I think should just work to create a symlink - I have not done it in a while. Can see it in practice at the nextjs repo for example (the `readme.md` file is a symlink): https://github.com/zeit/next.js.

Reply:
Skipping this for now as relative images need to be figured out.